src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1491,10 +1514,13 @@ private void InitializeNodeLibrary()
             foreach (var type in migrationTypes)
                 MigrationManager.AddMigrationType(type);
 
-            // Import Zero Touch libs
-            var functionGroups = LibraryServices.GetAllFunctionGroups();
             if (!IsTestMode)
+            {
```

Comment:
?

Reply:
The `functionGroups ` variable is used only inside the `if` condition, we needn't do extra work if it doesn't enter the `if` condition in the first place.